ctf_framework/templates/writeup/index.html

```diff
@@ -29,11 +29,4 @@ <h1>Writeups for <small>{{challenge}}</small></h1>
 
```

Comment:
Did some searches a few days ago, turns out you can do stuff like this in Django : 

https://stackoverflow.com/questions/34586259/how-to-organize-js-files-in-django

Maybe we should use that system instead of dumping everything in main.js?